bower.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "skrollr",
+  "version": "0.6.31",
```

Comment:
how is this related to this PR? The version is left out of bower on purpose as it can use tags.